.clang-format

```diff
@@ -5,7 +5,7 @@ AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
 AlignEscapedNewlinesLeft: true
 AlignOperands:   false
-AlignTrailingComments: false
+AlignTrailingComments: true
```

Comment:
I don't think we need really this. Having this option introduces bigger diffs which complicates merges... and the real value of having things aligned is kinda debatable.
This is quite unpopular setting across FB monorepo, so... I would prefer not to enable this.